timelock-server-benchmark-client/src/main/java/com/palantir/atlasdb/timelock/benchmarks/RandomBytes.java

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright 2017 Palantir Technologies, Inc. All rights reserved.
+ *
+ * Licensed under the BSD-3 License (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://opensource.org/licenses/BSD-3-Clause
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.palantir.atlasdb.timelock.benchmarks;
+
+import java.security.SecureRandom;
+
+public class RandomBytes {
+
+    private static final SecureRandom RANDOM = new SecureRandom();
```

Comment:
Do these need to be secure generated? Do we want `ThreadLocalRandom.current().nextBytes(result);`

Reply:
Don't need security but do want to more-or-less guarantee uniqueness. This is how UUID is implemented, and I wasn't sure how good the uniqueness guarantees were for `ThreadLocalRandom`